middleware/isAdmin.js

```diff
@@ -1,4 +1,5 @@
-const adminAccess = (req, res, next) => {
+class admin{
+const adminAccess (req, res, next) => {
```

Comment:
Why have you used the const keyword for your class function? Also give the function some spacing from the margin.